### ISSUE TITLE
consideration: overclocking for faster sampling?

### ISSUE THREAD

**Issue by hanetzer** [dotcypress/ula#6]
One can overclock the rp2040 sysclk to some fairly high numbers.
given that the pio runs at the sysclk rate, one could hypothetically get a much faster sampling rate.

**Comment by dotcypress** [comment]
Yes, this is possible, I tried 250 MHz and everything works fine. 
I need to do more tests on that and probably I will create an `overlocked` build
